pages_builder/generate_pages.py

```diff
@@ -87,7 +94,41 @@ def render_page(self, input_file):
                 partial['pageTitle'] +
                 " - Digital Marketplace frontend toolkit"
             )
-            if "examples" in partial:
+            if (
+                "examples" in partial
+            ):
+                template_file = (
+                    "toolkit/templates" +
+                    root.replace(self.pages_dirname, "") + "/" +
+                    file.replace(".yml", ".html")
+                )
+                if (os.path.isfile(template_file)):
+                    template = Template(
+                        open(template_file, "r").read()
+                    )
+                else:
+                    sys.exit("\n⚡ " + template_file + " not found ⚡")
+                for index, example in enumerate(partial["examples"]):
+                    parameters = dict(partial["examples"][index])
+                    if "title" in partial["examples"][index]:
```

Comment:
Could we have a comment here explaining why `title` is removed? It's a bit unclear from just reading the code.